Raven.Database/Storage/Esent/SchemaCreator.cs

```diff
@@ -788,33 +788,61 @@ public void CreateListsTable(JET_DBID dbid)
                 coltyp = JET_coltyp.DateTime,
                 grbit = ColumndefGrbit.ColumnNotNULL,
             }, null, 0, out columnid);
+           
+            var by_id = "+id\0\0";
+            var by_name_and_etag = "+name\0+etag\0\0";
+            var by_name_and_key = "+name\0+key\0\0";
+            var by_name_and_created_at = "+name\0+created_at\0\0";
 
-
-            CreateIndexes(tableid,
+            Api.JetCreateIndex2(session, tableid, new[]
+            {
                 new JET_INDEXCREATE
                 {
                     szIndexName = "by_id",
-                    szKey = "+id\0\0",
-                    grbit = CreateIndexGrbit.IndexPrimary
-                },
+                    cbKey = by_id.Length,
+                    szKey = by_id,
+                    grbit = CreateIndexGrbit.IndexDisallowNull | CreateIndexGrbit.IndexUnique,
```

Comment:
It appears to be an issue, you changed `CreateIndexGrbit.IndexPrimary` to `CreateIndexGrbit.IndexDisallowNull | CreateIndexGrbit.IndexUnique,`, was it intentional?

Reply:
I have no idea how esent indexes behave, I ran some tests they seemed to pass.
I also never written a schema upgrader.
I didn't mean to modify the behavior, I don't know what is the right grbit to provide...
Please tell me how should I delete the old indexes and create them correctly.
I think I can avoid deleting the primary index since it should not change, I just wanted all the indexes of the same table to have the same shape and trying to execute 4 indexes creation at once failed.